src/views/Project/index.jsx

```diff
@@ -0,0 +1,27 @@
+import { Component } from 'react';
+import Typography from 'material-ui/Typography';
+import projects from '../../data/loader';
+import BugsTable from '../../components/BugsTable';
+
+export default class Project extends Component {
+  state = {
+    fileName: this.props.match.params.projectKey,
+    projects,
+  };
+  render() {
+    const { fileName, projects } = this.state;
+    const projectInfo = projects[fileName];
+
+    return (
+      <div>
+        <header>
```

Comment:
I remember we removed the top level `div`. I think something went wrong during the rebase in this file.